pages.es/common/git-rev-list.md

```diff
@@ -9,7 +9,7 @@
 
 - Muestra los commits más reciente a partir de una fecha y una rama específica:
 
-`git rev-list --since={{'2019-12-01 00:00:00'}} {{master}}`
+`git rev-list --since={{'2019-12-01 00:00:00'}} {{branch_name}}`
```

Comment:
Same as [above](https://github.com/tldr-pages/tldr/pull/4533/files#r500594375) — this should be translated.

Reply:
```suggestion
`git rev-list --since={{'2019-12-01 00:00:00'}} {{nombre_de_rama}}`
```